cordova-lib/src/configparser/ConfigParser.js

```diff
@@ -417,11 +402,11 @@ ConfigParser.prototype = {
     getEngines: function(){
```

Comment:
Please, revert the changes to 'getEngines'. they screw up the indenting and they are a no-op.